src/bin/tikv-server.rs

```diff
@@ -789,7 +829,12 @@ fn main() {
     let pd_client = RpcClient::new(&pd_endpoints).unwrap();
     let cluster_id = pd_client.cluster_id;
 
-    let mut cfg = build_cfg(&matches, &config, cluster_id, addr);
+    let mut system = System::new();
+    system.refresh_all();
```

Comment:
No need to `refresh_all` again.

https://github.com/GuillaumeGomez/sysinfo/blob/37829411d4186264c24dfa96f15522655d5060bb/src/linux/system.rs#L70